src/main/java/godaddy/registry/jtoolkit2/session/TLSContext.java

```diff
@@ -27,9 +27,9 @@
  * </p>
  */
 public class TLSContext {
-    private static final String TLSV1 = "TLSv1";
+    protected static final String TLSV1 = "TLSv1";
```

Comment:
Yes, I did used `TLSV1` in child version of `TLSContext`. Is it ok ?? Or should I use my own local constant and revert this back to private ?

Reply:
It would be nice to define its own local constants.

In the future if there is any change made in `TLSContext` (e.g. this constant may get removed for some reason, say, need to change the toolkit to force use different protocol), it would not be a surprise to the subclass (as a compilation error).

Reply:
Sure, defined locally.